### PR TITLE
SEO-176113-MVC-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the missing H1

Regards,
Beryl.

### CODE REVIEW

File: aspnetmvc/DataManager/Antiforgery.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ For this purpose, the input element with hidden value field and name attribute i
       }
 {% endhighlight %}
 
-![](Antiforgery_images/Antiforgery.png)
+![Antiforgery Token in ASP.NET MVC](Antiforgery_images/Antiforgery.png)
```
Comment (Yvone-Atieno) [inline aspnetmvc/DataManager/Antiforgery.md:84]: Should include control name also

File: aspnetmvc/DataManager/Antiforgery.md [hunk 2]
```diff
@@ -81,8 +81,8 @@ For this purpose, the input element with hidden value field and name attribute i
...
 
 In the header, You can find the anti-forgery token value
 
-![](Antiforgery_images/Antiforgery_header.png)
+![Antiforgery Token Value in ASP.NET MVC](Antiforgery_images/Antiforgery_header.png)
```
Comment (Yvone-Atieno) [inline aspnetmvc/DataManager/Antiforgery.md:88]: Should include control name

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnetmvc/DataManager/Antiforgery.md (hunk 1 above)
Comment (berylouma) [reply aspnetmvc/DataManager/Antiforgery.md]: Added

File: aspnetmvc/DataManager/Antiforgery.md (hunk 2 above)
Comment (berylouma) [reply aspnetmvc/DataManager/Antiforgery.md]: Added

**Pull request merged**